hugo/content/gettingstarted/Design/tls.md

```diff
@@ -8,25 +8,71 @@ weight: 6
 ## TLS Configuration
 
 Should you desire to alter the default TLS settings for the Postgres Operator, you can set the
-following variables in bash:
+following variables as described below.
+
+### Server Settings
 
 To disable TLS and make an unsecured connection on port 8080 instead of connecting securely over
 the default port, 8443, set:
 
 Bash environment variables    
+```bash
```

Comment:
I'd recommend putting some space between the paragraphs + the coding blogs, e.g.:

```
Bash environment variables

``bash
DISABLE_TLS=true
``
```

Reply:
Added the extra spaces. Verified that this did not impact the formatting when built with Hugo.

Reply:
Added space between sections, verified that Hugo renders the page as expected after changes.